web/packages/teleport/src/Account/Account.tsx

```diff
@@ -237,7 +237,7 @@ export function Account({
                   }
                   description="Provide secondary authentication when signing in
                     with a password. Unlike passkeys, multi-factor methods do
-                    not enable passwordless sign-in."
+                    not enable passwordless sign-in. Multi-factor methods are usable for required multi-factor authentication checks for sessions."
```

Comment:
Maybe skip this change? I don't think we had confusion in this area before.

Reply:
removed the mfa desc change